services/thermal_mgr/thermal_mgr.c

```diff
@@ -41,20 +41,57 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 }
 
+#define MAX_MS_WAIT 20
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+  xQueueSend(thermalMgrQueueHandle, (void *)event, portTICK_PERIOD_MS*MAX_MS_WAIT);
```

Comment:
check if the queueSend succeeded or timed out

Reply:
Modified to check for pdTRUE